Birthday-2024-Project/Art/SpoofedSplash.png.import

```diff
@@ -0,0 +1,34 @@
+[remap]
```

Comment:
game always starts in full screen and then adjust to the settings actually used by user instead of starting in already requested resolution